saleor/checkout/base_calculations.py

```diff
@@ -26,19 +25,8 @@ def calculate_base_line_unit_price(
     line_info: "CheckoutLineInfo",
     channel: "Channel",
     discounts: Optional[Iterable[DiscountInfo]] = None,
-) -> CheckoutPricesData:
-    """Calculate line unit prices including discounts and vouchers.
-
-    Returns a three money values. Undiscounted price, price and price with voucher.
-    Voucher is added to 'price_with_discounts' when line's product matches to products
-    applicable for voucher.
-    For voucher with 'apply once per order', the price will be reduced in calculation
-    of total price.
-    'price' includes discount from sale if any valid exists.
-    'price_with_discounts' includes voucher discount and sale discount if any valid
-    exists.
-    'undiscounted_price' is a price without any sale and voucher.
-    """
+) -> Money:
```

Comment:
Would be good to have some description here. Especially that we're applying here voucher

Reply:
Somethin like:
```
"""Calculate line unit price.
Unit price includes discount from sale and voucher
A voucher is added to the unit price when line's product matches to products applicable for the voucher.
"""
```